src/Controller/PagesController.php

```diff
@@ -42,7 +42,9 @@ public function display(...$path)
     {
         $count = count($path);
         if (!$count) {
-            return $this->redirect('/');
```

Comment:
I see. Then the return type ``void`` is wrong. https://github.com/cakephp/app/pull/511/files#diff-187a1f16b006225e2ca35de472fc4641R36

Then it would be ``null``, as it would fit the return type of  ``Controller::redirect()`` this way.
https://api.cakephp.org/3.4/class-Cake.Controller.Controller.html#_redirect